kaitvan/front-end-capstone
# Activities in array appear on DOM

## Description
Added an activity bubble component and all activities held in the activities array in state are displayed on the DOM.

## Related Issue
Resolves #4 

## Motivation and Context
This change allows the user to see all activities previously added to their self-care list.

## How Can This Be Tested?
```git fetch origin development```
If you change any of the items listed in the activities array on the MyList view, each item should appear in its own blurb on the DOM.

## Screenshots (if appropriate):
![Screen Shot 2020-12-06 at 9 22 24 AM](https://user-images.githubusercontent.com/63985074/101284235-97731d00-37a4-11eb-89e8-974b95654dce.png)

## Types of changes
